simpleGravitySim/simulation.jl

```diff
@@ -0,0 +1,79 @@
+#modules and packages
+include("./simpleGravity.jl")
+using .simpleGravity
+using Trixi
+using OrdinaryDiffEq
+using Plots
+using Printf
+
+
+function build_initial_conditions(user_supplied_ρ::Function)
+    # define named lambda
+    function _init_conditions(x, t, equations)
+        # call user supplied function
+        ρ = user_supplied_ρ(x, t, equations)
+        #using the acceleration calculation in the og paper's setup for now
+        q1 = -2*π*sin(π * x[1])
+        return SVector(ρ,q1)
+    end
+    # closures are captured, so return the function to use in trixi
+    return _init_conditions
+end
+
+#set up solver for equations
+function setSolver(initCond, equations, mesh)
+    boundary_conditions = (x_neg=BoundaryConditionDirichlet(((x, t, equations) -> SVector(sin(x[1]), -2*π*sin(π * x[1])))),
+                       x_pos=BoundaryConditionDirichlet(((x, t, equations) -> SVector(sin(x[1]), -2*π*sin(π * x[1])))))
```

Comment:
Try just using `Trixi.boundary_condition_do_nothing`. We can chat about why this might be causing issues on Thursday.